MOBILE_API.xml

```diff
@@ -4081,7 +4146,10 @@
             <description>If included, the platform supports RC climate controls. For this baseline version, maxsize=1. i.e. only one climate control module is supported.</description>
         </param>
         <param name="radioControlCapabilities" type="RadioControlCapabilities" mandatory="false" minsize="1" maxsize="100" array="true">
-            <description>If included, the platform supports RC radio controls.For this baseline version, maxsize=1. i.e. only one radio control module is supported.</description>
+            <description>
+                If included, the platform supports RC radio controls.For this baseline version, maxsize=1. i.e. only one radio control module is supported.
```

Comment:
```suggestion
                If included, the platform supports RC radio controls.
```

This no longer is true, right?

Reply:
That's correct, fixed in https://github.com/smartdevicelink/rpc_spec/pull/290/commits/bdbb236b620e46efb451cf4204aa0c955f9598b6